src/i18n/en-EU.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file will be used with the `defaultLocale` is set to `en-EU`, making it use the different date format. For all other strings, it will default to the fallback language.